services/brig/src/Brig/Options.hs

```diff
@@ -589,7 +589,10 @@ data Settings = Settings
     setEnableDevelopmentVersions :: Maybe Bool,
     -- | Minimum delay in seconds between consecutive attempts to generate a new verification code.
     -- use `set2FACodeGenerationDelaySecs` as the getter function which always provides a default value
-    set2FACodeGenerationDelaySecsInternal :: !(Maybe Int)
+    set2FACodeGenerationDelaySecsInternal :: !(Maybe Int),
+    -- | The time-to-live of a nonce in seconds.
+    -- use `set2FANonceTtlSecs` as the getter function which always provides a default value
```

Comment:
comment needs more editing.

Reply:
Can explain what how this needs to be edited?